request.go

```diff
@@ -303,7 +303,9 @@ func (r *Request) SetBody(body interface{}) *Request {
 // Accessing a result value from response instance.
```

Comment:
I wonder if this should be documented here? something like
`passing a nil res paramaeter has no effect`

Reply:
@segevda I will add it later it. Thanks!